src/test/user.spec.js

```diff
@@ -674,3 +675,115 @@ describe('PATCH /api/v1/auth/user/:id/role', () => {
     expect(userRoleStatus.length).to.equal(1);
   });
 });
+
+describe('/api/v1/auth/reset-password', async () => {
+  await db.User.create({
+    username: 'mikey',
+    password: '1234567890',
+    email: 'olorunwalawrence5@gmail.com'
+  });
+  it('it should send password reset link to the email provided if it is associated with a registered email', done => {
+    const token = helper.jwtSigner(600, {
+      email: 'olorunwalawrence5@gmail.com'
+    });
+    chai
+      .request(app)
+      .post('/api/v1/auth/reset-password')
+      .set('x-access-token', token)
+      .send({
+        email: 'olorunwalawrence5@gmail.com'
+      })
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body).to.have.property('message');
+        expect(res.body.message).to.be.equal(
+          'A password reset link would be sent to the email provided if it is associated with a registered email'
+        );
+      });
+
+    done();
+  });
+
+  it('should return invalid email credential', done => {
+    const token = helper.jwtSigner(600, {
+      email: 'olorunwalawrence5@gmail.com'
+    });
+    chai
+      .request(app)
+      .post('/api/v1/auth/reset-password')
+      .set('x-access-token', token)
+      .send({
+        email: '1example@gmail.com'
+      })
+      .end((err, res) => {
+        expect(res.status).to.equal(401);
+        expect(res.body).to.have.property('message');
```

Comment:
And you want to also remove this assertion for `property` test. I don't think it's necessary.